VMBackup/main/handle.py

```diff
@@ -195,6 +200,11 @@ def freeze_snapshot(timeout):
         if(hutil.get_value_from_configfile('seqsnapshot') == None):
             hutil.set_value_to_configfile('seqsnapshot', '0')
         seqsnapshotflag = hutil.get_value_from_configfile('seqsnapshot')
```

Comment:
Same againg. seqsnapshot is not being used here. As a cleanup you can remove this logic.